codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -185,7 +195,14 @@ public final T build(Http2Connection connection) {
                 reader = new Http2InboundFrameLogger(reader, frameLogger);
                 writer = new Http2OutboundFrameLogger(writer, frameLogger);
             }
+
+            // Configure the flow controllers.
+            connection.local().flowController(new DefaultHttp2LocalFlowController(connection, writer,
+                    DEFAULT_WINDOW_UPDATE_RATIO, autoRefillConnectionWindow));
+            connection.remote().flowController(new DefaultHttp2RemoteFlowController(connection));
```

Comment:
Yes good point ... setting the local flow controller would be difficult either way with the current builder. With the current type relationships...the first thing that comes to mind is a setter to set the `Http2FrameWriter` on the local flow controller.